assets/constants.json

```diff
@@ -7,5 +7,6 @@
     "Mathematics", "Medicine", "Music", "Philosophy", "Physics", "Poetry",
     "Probability", "Programming", "Puzzles", "Reading", "Spanish", "Sport",
     "Statistics", "Trigonometry", "Welcome"],
-  "DISABLED_EXPLORATION_IDS": ["5"]
+  "DISABLED_EXPLORATION_IDS": ["5"],
+  "NEW_STATE_TEMPLATE":{"content":[{"type":"text","value":""}],"interaction":{"answer_groups":[],"confirmed_unclassified_answers":[],"customization_args":{"rows":{"value":1},"placeholder":{"value":"Type your answer here."}},"default_outcome":{"dest":"(untitled state)","feedback":[],"param_changes":[]},"fallbacks":[],"id":"TextInput"},"param_changes":[]}
```

Comment:
Ah, OK, I think I see. Is A used elsewhere, other than for defining B? If not, we might as well get rid of A.

If A is used elsewhere, then ... I'm not sure there's a good solution. The best I can think of is to have a service (that can be called from both main code and test code) that returns the value of B. Then the logic for the construction of B would happen only in one place (i.e. in the main code).